templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try).
+
+If you try either package, <<feedback, let us know what you think>>.
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. We've also made excellent TCK progress with these features.
+// // // // // // // //
+// ABOVE:
+// Update the second and third sentence appropriately for this release
+// // // // // // // //
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* ITEM_1
+* ITEM_2
+* ITEM_3
+// // // // // // // //
+// ABOVE:
+// Change the ITEMS above to list the new changes and features in this
+// beta release
+// // // // // // // //
+
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+// // // // // // // //
+// Copy feature from both lists in the previous beta post creating    // a cumulative list from all previous betas to make a list of all    // existing features in the new beta.
+// // // // // // // //
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.RELEASE_VERSION-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.RELEASE_VERSION-beta,)'
+}
+----
+// // // // // // // //
+// Above:
+// Replace RELEASE_VERSION with the version number of Open Liberty
+// // // // // // // //
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want (but remember if you enable the `jsp-3.0` feature, the `servlet-5.0` and `el-4.0` features are automatically enabled for you) or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
```

Comment:
Remove the "but remember..." parentheses.

Reply:
Done